pkg/rancher-desktop/assets/lima-config.yaml

```diff
@@ -44,6 +44,13 @@ provision:
     #!/bin/sh
     set -o errexit -o nounset -o xtrace
     fstrim /mnt/data
+- # allow more than 10 sessions over the master control path
+  mode: system
+  script: |
+    #!/bin/sh
+    set -o errexit -o nounset -o xtrace
+    sed -i 's/#MaxSessions 10/MaxSessions 25/g' /etc/ssh/sshd_config
```

Comment:
Hmm, I'm trying to decide if we need to match existing uncommented lines too… `/^#?MaxSessions [0-9]+/` sort of thing.  In case we end up having a number that's too small (but that'd mostly be when we bump it next time, so it might be okay?)

Reply:
I'll create another PR for that later; I want this fix in "right now" 😄